### PR TITLE
style(prod and capability and feature): capitalization and usage guide

### PR DESCRIPTION
**Pull request by mmfred**
This is the capitalization guide that's been compiled by people across NR.

### CODE REVIEW

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+                IO
+                I/O</td>
+        </tr>
+        <tr>
+            <td>New Relic Lookout</td>
+            <td>capability of New Relic One</td>
+            <td>"First use: New Relic Lookout
+                Subsequent uses: New Relic Lookout
+                Describing actions in the UI: Lookout</td>
+            <td>New Relic Lookout’s
```
Comment (roadlittledawn) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:161]: i'd recommend replacing these less common slanted apostrophes with single quote. if you're using VSCode it highlights them with a little light bulb to let you know.

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>What it is</th>
+            <th>✔ Use this</th>
+            <th>✖ Not this</th>
+        </tr>
+    </thead>
+    <tbody>
```
Comment (roadlittledawn) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:38]: to fix the error, try adding an empty line between `</thead>` and `<tobdy>`

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            <td>"First mention: New Relic, Inc. (corporation/entity), New Relic® (printed assets), or New Relic (digital assets)
+            Subsequent mentions: New Relic, our company, we, or it
+            </td>
+            <td>"New Relic’s
+                new relic
+                New relic
+                NR 
+                their
+            </td>
+        </tr>
```
Comment (roadlittledawn) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:51]: and possibly empty lines between opening and closing `<tr>` elements

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 1 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:161]: done.

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+                Subsequent uses: New Relic One or New Relic One platform
+            </td>
+            <td>New Relic One’s
+                New Relic one
+                NR1
+            </td>
+        </tr>
+        <tr>
+            <td>FutureStack*
+                {Future}Stack*
```
Comment (paperclypse) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:65]:
```suggestion
                {Future}Stack*</td>
```

**Comment by mmfred** [comment]
Hi hero - I'm ready to move this page to release!

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 5]
```diff
@@ -0,0 +1,638 @@
...
+
+New Relic trademarks are adjectives (brand names) modifying nouns (the generic product type).
+* Do not make New Relic or its platform, capabilities, and so on possessive using apostrophes. For example, use the “functionality of New Relic One” or “the New Relic One functionality” instead of “New Relic One's functionality.”
+* Do not pluralize New Relic or its platform, capabilities, and so on. 
+* Do not abbreviate or combine New Relic or its platform, capabilities, and so on. 
+* Do not hyphenate New Relic or its platform, capabilities, and so on, and do not allow them to break across a page line when used in text. 
+* Avoid confusion by clearly separating and identifying New Relic trademarks from other companies' names and/or trademarks. 
+These trademark guidelines apply to other companies' trademarked names and products too.
+
+## Naming
```
Comment (zuluecho9) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:22]: Needs some links to the sections here.

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 6]
```diff
@@ -0,0 +1,638 @@
...
+            <td>First use: NerdGraph® (printed assets) or NerdGraph (digital assets)
+                
+                Subsequent uses: NerdGraph</td>
+            <td>Nerd Graph, Nerdgraph, nerdgraph, nerd graph
+            </td>
+        </tr>
+
+        <tr>
+            <td>Nerdlet*</td>
+            <td>component that make up New Relic One apps; a specific UI view represented by a React JavaScript package</td>
```
Comment (zuluecho9) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:95]: few of these have 'component that make up...', maybe should be 'a component of New Relic One apps'...

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 7]
```diff
@@ -0,0 +1,638 @@
...
+                Subsequent uses: Infinite Tracing
+            </td>
+            <td>Infinite tracing, infinite tracing
+            </td>
+        </tr>
+        
+        <tr>
+            <td>New Relic Explorer</td>
+            <td>capability of New Relic One</td>
+            <td>First use: New Relic explorer
```
Comment (zuluecho9) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:146]: should be 'New Relic Explorer'

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 8]
```diff
@@ -0,0 +1,638 @@
...
+                
+                Subsequent uses: New Relic Explorer
+                
+                Describing actions in the UI: Explorer</td>
+            <td>New Relic Explorer's
```
Comment (zuluecho9) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:151]: Not sure if 'New Relic Explorer's Explorer' is a typo

Comment (zuluecho9) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:151]: Oh I see. These are like different entries. this one was just lacking a comma like the others

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx [hunk 9]
```diff
@@ -0,0 +1,638 @@
...
+            <td>feature of the log management capability in New Relic One; tracing logs throughout a complex service</td>
+            <td>logs in context</td>
+            <td>Logs in Context, Logs in context
+            </td>
+        </tr>
+        
+        <tr>
+            <td>metrics, events, logs, and traces</td>
+            <td>what you monitor for full-stack observability</td>
+            <td>First use: metrics,events,logs, and traces or metrics, events,logs, and traces (MELT)
```
Comment (zuluecho9) [inline src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:445]: some unusual comma spacing here

**Review by zuluecho9 — CHANGES_REQUESTED** [review]

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 5 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:22]: I took out the naming section.  thanks.

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 6 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:95]: fixed!

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 7 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:146]: fixed!

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 8 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:151]: fixed.

File: src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx (hunk 9 above)
Comment (mmfred) [reply src/content/docs/style-guide/word-choice/product-capability-feature-usage.mdx:445]: fixed.

**Pull request merged**